ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 5.5.0f3
+m_EditorVersion: 5.5.1f1
```

Comment:
Ooooh at runtime, sorry I missed that part D:  Yes there is no support for changing the hierarchy at runtime, which includes enabling/disabling the anchors of constant size.